quality-control/human-clinical-and-health-data.md

```diff
@@ -1,27 +1,79 @@
+ 
+* Software Development Life Cycle:
+Following a software development life cycle, including quality assurance systems, is crucial. Adhering to ISO [250xx](https://iso25000.com/index.php/en/iso-25000-standards) standards ensures appropriate design, development, and testing of software. Additionally, guidelines like the [EMA/226170/2021](https://www.ema.europa.eu/en/documents/regulatory-procedural-guideline/draft-guideline-computerised-systems-and-electronic-data-clinical-trials_en.pdf) provide direction for implementing computerized systems and electronic data in clinical trials.
+
+* International Organization for Standardization (ISO) Standards:
+ISO has produced various standards offering frameworks for different data management aspects. [ISO 9000](https://www.iso.org/standard/45481.html) focuses on quality management systems, [ISO 8000](https://www.iso.org/standard/81745.html) on Master Data quality and exchange, and [ISO 25012](https://www.iso.org/standard/35736.html) provides a general data quality model applicable across the data life cycle.
+ 
+* [Semantic Web Principles](https://ieeexplore.ieee.org/document/6337108):
+Applying semantic web principles is essential, emphasizing factors such as the availability, accessibility, and reliability of data sources, absence of noise and inconsistencies in raw data, and the use of high-quality validated vocabularies for semantic conversion.
+ 
+* [World Health Organization (WHO) Data Quality Principles](https://cdn.who.int/media/docs/default-source/data-quality-pages/2021_dqa_module-1-framework-and-metrics-19-04-21.pdf?sfvrsn=13c95fb1_3&sequence=1&isAllowed=y):
+WHO outlines principles such as accuracy, validity, reliability, completeness, readability, timeliness, punctuality, accessibility, usefulness, confidentiality, and security for ensuring data quality. WHO has produced the [Data Quality Assurance (DQA)](https://www.who.int/data/data-collection-tools/health-service-data/data-quality-assurance-dqa) toolkit to support countries in assessing and improving the quality of Routine Health Information Systems (RHIS) data.
+
+* [Standardization in Digital Health Europe](https://digitalhealtheurope.eu/results-and-publications/digital-health-standards/):
+Digital Health Europe defines four categories for standardizing health data: information model, vocabulary/terminology standards, content standards, and communication standards. Standardization is considered fundamental for creating high-quality data.
+
+* [Clinical Trial Registration](https://www.jmir.org/2023/1/e41446):
+The importance of clinical trial registration is emphasized for ensuring accountability and transparency in clinical research. Timely and accurate data on trial registries are crucial for evaluating reported studies. 
```

Comment:
Clinical trial registration ensures accountability and transparency in clinical research. Timely and accurate data on trial registries are crucial for evaluating reported studies. 

It was unclear who emphasized it